services/npm/npm-collaborators.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const { colorScale } = require('../../lib/color-formatters')
+const NpmBase = require('./npm-base')
+
+const keywords = ['node']
+
+const collaboratorColor = colorScale([2, 3, 4, 6])
```

Comment:
Low bus factor is good, but I'm not totally convinced on trying to devise an absolute scale of what number of collaborators is 'good' or 'bad' and applying that to all projects regardless of the size of their codebase or community.

I think its reasonable to say only one maintainer is bad (and so is zero) for all projects, but maybe just report the number in blue for everything >1 ?

Reply:
Part of my use case is discerning the size of the community, so it’s a little chicken and egg. I can see your point that we’re not in a position to say that 4 is too few or 6 is enough, and that the expectation of what’s good would change for widely used tools.

1 is bad. Should 2 be caution? (Maybe orange or yellow.) Keeping in mind a badge this administrative is less likely to be used on individual readmes than on organizational badgeboards or project comparison scorecards tracking a large number of projects.